controllers/articles.controller.js

```diff
@@ -10,3 +11,13 @@ exports.getArticle = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticles = (req, res, next) => {
+  fetchArticles()
+    .then((rows) => {
+      res.status(200).send({ articles: rows });
```

Comment:
The variable name rows is not an appropriate name for this. I would recommend using something like articles instead or something more descriptive.